android/libraries/rib-compiler-app/src/test/java/com/uber/rib/compiler/InteractorAnnotationVerifierTest.java

```diff
@@ -20,6 +20,7 @@
 import static com.google.common.truth.Truth.assert_;
 import static com.google.testing.compile.JavaSourcesSubjectFactory.javaSources;
 
+// TODO: fix failing tests
```

Comment:
Yeah, let's do it ;)

Reply:
This module is turned off in settings.gradle, will probably be removed as well